tests/foreman/api/test_subscription.py

```diff
@@ -311,7 +310,9 @@ def test_sca_end_to_end(module_ak, rhel7_contenthost, module_org, rh_repo, custo
 
 
 @pytest.mark.tier2
-def test_positive_candlepin_events_processed_by_stomp(rhel7_contenthost, function_org, target_sat):
+def test_positive_candlepin_events_processed_by_stomp(
+    rhel7_contenthost, function_entitlement_manifest, function_org, target_sat
```

Comment:
```suggestion
    rhel7_contenthost, function_entitlement_manifest_org, target_sat
```

Reply:
At first I implemented this change, then, while debugging locally, I realized that the change actually breaks the test. The first bits of the test are registering a content host to an organization that has not had a manifest uploaded yet and verifying that the host does not have a valid subscription. Then, the test uploads a manifest, attaches a subscription to the host, and verifies that it does have a valid subscription. So we actually need to use separate organization and manifest fixtures in this case.